pkg/driver/distro/distro.go

```diff
@@ -189,48 +192,58 @@ func Build(ctx context.Context,
 	driverType drivertype.DriverType,
 	driverVer string,
 ) (string, error) {
+	printer.Logger.Info("Trying to compile the requested driver")
 	driverFileName := toFilename(d, &kr, driverName, driverType)
-	destination := toLocalPath(driverVer, driverFileName, kr.Architecture.ToNonDeb())
-	if exist, _ := utils.FileExists(destination); exist {
-		return destination, ErrAlreadyPresent
+	destPath := toLocalPath(driverVer, driverFileName, kr.Architecture.ToNonDeb())
+	if exist, _ := utils.FileExists(destPath); exist {
+		return destPath, ErrAlreadyPresent
 	}
 
 	env, err := d.customizeBuild(ctx, printer, driverType, kr)
 	if err != nil {
 		return "", err
 	}
 
-	if env == nil {
-		env = make(map[string]string)
-	}
+	srcPath := fmt.Sprintf("/usr/src/%s-%s", driverName, driverVer)
+
+	// TODO shall we call fixupkernel here?
+	fixedKr := d.FixupKernel(kr)
+	ro := cmd.NewRootOptions()
+	ro.Architecture = fixedKr.Architecture.String()
+	ro.DriverVersion = driverVer
+	ro.KernelVersion = fixedKr.KernelVersion
+	ro.ModuleDriverName = driverName
+	ro.ModuleDeviceName = driverName
+	ro.KernelRelease = fixedKr.String()
+	ro.Target = d.String()
+	ro.Output = driverType.ToOutput(destPath)
+
+	// Setup slog logger (used by driverkit)
+	// to log on our logger level and writer.
+	programLevel := &slog.LevelVar{}
```

Comment:
This will probably be moved to its own helper function.
Basically we want to enforce `slog` logger (used by driverkit) to log at our own log level on our own log writer.